components/environment/build.gradle.kts

```
@@ -0,0 +1,29 @@
+plugins {
+  `java-library`
+  id("com.gradleup.shadow")
+}
+
+apply(from = "$rootDir/gradle/java.gradle")
+
+/*
+ * Add an addition gradle configuration to be consumed by bootstrap only.
+ */
+tasks.shadowJar {
+  relocate("datadog.environment", "bootstrap.environment")
```

`bootstrap.environment` is not a known excluded package for jacoco instrumentation, this cause the `....core.internal.instr.ClassInstrumenter` to fail.

Relocating under a known excluded prefix solves the issue.

```suggestion
  relocate("datadog.environment", "datadog.trace.bootstrap.environment")
```

FYI `datadog.trace.bootstrap` is already used here for relocation purpose as well.

https://github.com/DataDog/dd-trace-java/blob/3dc0c20c31408d6ee23df6aab6a68fe51bb31d8f/dd-java-agent/build.gradle#L81-L81